test/cppsuite/test_harness/workload/thread_context.h

```diff
@@ -114,13 +127,30 @@ class transaction_context {
     bool _in_txn = false;
 };
 
+enum thread_type { READ, INSERT, UPDATE };
+
+static std::string
+type_string(thread_type type)
+{
+    switch (type) {
+    case thread_type::INSERT:
+        return ("insert");
+    case thread_type::READ:
+        return ("read");
+    case thread_type::UPDATE:
+        return ("update");
+    default:
+        testutil_die(-1, "unexpected thread_type");
```

Comment:
Should we print the value of type ?

Reply:
No because this is the function that prints the value of the type.

Reply:
static_cast<int>(type) would be fine

Reply:
5abf1d1c6